apps/ledger-live-mobile/src/components/RootNavigator/BaseNavigator.tsx

```diff
@@ -553,6 +548,11 @@ export default function BaseNavigator() {
           options={{ ...TransparentHeaderNavigationOptions, title: "" }}
           component={RedirectToOnboardingRecoverFlowScreen}
         />
+        <Stack.Screen
```

Comment:
that was just a temp change I made to move them closer together in the base navigator so that i could directly compare them in the file easier, but then i decided just to keep it because they are so heavily related and it seems more useful if they are closer